addons/sourcemod/scripting/include/ttt.inc

```diff
@@ -175,7 +179,7 @@ forward Action TTT_OnClientDeathPre(int victim, int attacker);
  * @param ragdoll           The Array with Ragdoll informations (by reference).
  * @param silentID          True if body was silenced id and false if normal inspect
  */
-forward void TTT_OnBodyFound(int client, int victim, int[] ragdoll, bool silentID);
+forward void TTT_OnBodyFound(int client, int victim, Ragdolls ragdoll, bool silentID);

```

Comment:
This will not work, you can't push/use enum struct in forwards. I already tried but found another workaround.